ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5655,4 +5655,120 @@ RFVSC2X:
+  - water_riser_mode
+  - cooling_output_percentage_command
+  - heating_output_percentage_command
+
+DPHC4:
+  description: "Four pipe heating and cooling controls for radiant panels"
+  is_abstract: true
+  opt_uses:
+  - supply_water_isolation_valve_status
+  - return_water_isolation_valve_status
+  - heating_supply_water_temperature_setpoint
+  - chilled_supply_water_temperature_setpoint
+  uses:
+  - heating_supply_water_temperature_sensor
+  - heating_return_water_temperature_sensor
+  - chilled_supply_water_temperature_sensor
+  - chilled_return_water_temperature_sensor
+  - cooling_output_percentage_command
+  - heating_output_percentage_command
+
+HHCD42:
+  description: "Hydronic heating and cooling distribution monitoring for change over group with a heat and a cold exchanger. Primary side: Four-pipe installation 
+                with flow control valves for cooling and heating. Two pipe installation with isolation valves on the secondary side towards the radiant panels. 
+                Heating or cooling only"
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - heating_supply_water_isolation_valve_status
+  - heating_return_water_isolation_valve_status
+  - chilled_supply_water_isolation_valve_status
+  - chilled_return_water_isolation_valve_status
+  - chilled_water_valve_percentage_command
+  - heating_water_valve_percentage_command
+  - secondary_heating_supply_water_temperature_sensor
+  - secondary_heating_return_water_temperature_sensor
+  - secondary_chilled_supply_water_temperature_sensor
+  - secondary_chilled_return_water_temperature_sensor
+  - heating_request_count
+  - cooling_request_count
+  opt_uses:
+  - average_zone_air_temperature_sensor
+  - primary_chilled_supply_water_temperature_sensor
+  - primary_heating_supply_water_temperature_sensor
+
+HHCD44:
+  description: "Hydronic heating and cooling distribution monitoring for change over group with a heat and a cold exchanger. Primary side: Four-pipe installation 
+                with flow control valves for cooling and heating. Four pipe installation with isolation valves on the secondary side towards the radiant panels.
+                Heating and cooling in parallel"
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - heating_supply_water_isolation_valve_status
+  - heating_return_water_isolation_valve_status
+  - chilled_supply_water_isolation_valve_status
+  - chilled_return_water_isolation_valve_status
+  - chilled_water_valve_percentage_command
+  - heating_water_valve_percentage_command
+  - secondary_heating_supply_water_temperature_sensor
+  - secondary_heating_return_water_temperature_sensor
+  - secondary_chilled_supply_water_temperature_sensor
+  - secondary_chilled_return_water_temperature_sensor
+  - heating_request_count
+  - cooling_request_count
+  opt_uses:
+  - average_zone_air_temperature_sensor
+  - primary_chilled_supply_water_temperature_sensor
+  - primary_heating_supply_water_temperature_sensor
+
+DFVSCS:
```

Comment:
This is not necessary, we should be using DFVSC. If you have a type which does not have the command but merely the sensor, use a MISSING field for the command in the translation (but I would question why we cannot control the fan speed).

Reply:
Understood. I'll remove it from the pr.